### PR TITLE
Ond211 1724 beautify exported agents

### PR DESCRIPTION
**Pull request by JSteinbauer**
Adds a field specifying the compression_level to the ExportAgentRequest message.

### CODE REVIEW

File: ondewo/nlu/agent.proto [hunk 1]
```diff
@@ -413,6 +413,10 @@ message ExportAgentRequest {
     // Note: The URI must start with
     // "gs://". If left unspecified, the serialized agent is returned inline.
     string agent_uri = 2;
+
+    // Optional. The compression level of the zip file created.
+    // Valid range 1-9; other values are replaced by the default value 1
```
Comment (gaborod16) [inline ondewo/nlu/agent.proto:418]: oh, so, we are not raising an error but amending the input value?

File: ondewo/nlu/intent.proto [hunk 2]
```diff
@@ -968,4 +971,30 @@ message DeleteTrainingPhraseRequest {
     // Required. The name of the training phrase.
     // Format: `projects/<Project ID>/agent/intents/<Intent ID>/trainingPhrases/<Training Phrase ID>`
     string name = 1;
+}
+
+// The request message for TrainingPhraseRequest
+message ListTrainingPhraseRequest {
```
Comment (gaborod16) [inline ondewo/nlu/intent.proto:977]: Careful.
These changes do not belong to this PR.

Make sure to merge the changes from the other PR when resolving conflicts (if the situation arises)

**Review by gaborod16 — APPROVED** [review]

File: ondewo/nlu/agent.proto (hunk 1 above)
Comment (JSteinbauer) [reply ondewo/nlu/agent.proto:418]: I think you're right, raising an error would be better. I'll make a change.

File: ondewo/nlu/intent.proto (hunk 2 above)
Comment (JSteinbauer) [reply ondewo/nlu/intent.proto:977]: Indeed, I'll take care of that.
